K2InformaticsGmbH/dderl
# Updated front end libs

Fixes #414 
Updated webpack, jquery, jquery-ui and d3, tested with latest stable nodejs 6.10.2 and npm 3.10.10
To test:
Download nodejs 6.10.2 from https://nodejs.org/en/ it includes npm 3.10.10, verify the version using: ```
$ node -v
v6.10.2
$ npm -v
3.10.10
```
delete ``node_modules`` directory from dderl/priv/dev and run:
```
npm install
npm run build
```
Verify dderl looks the same as before (Working hard for it to be the case 😄 )
